README.md

```diff
@@ -64,30 +58,27 @@ sudo snap connect openthread-border-router:bluetooth-control
 sudo snap connect openthread-border-router:bluez
 ```
 
-> **Note**  
-> On **Ubuntu Core**, the `avahi-control` and `bluez` interfaces are not provided by the system.
-> These interfaces should be consumed from other snaps, such as the [Avahi](https://snapcraft.io/avahi) and [BlueZ](https://snapcraft.io/bluez) snaps.
-> 
-> To install the snaps, and establish connections for the `avahi-control` interface from the `avahi` snap, and the `service` interface from the `bluez` snap, run:
-> ```bash
-> sudo snap install avahi bluez
-> sudo snap connect openthread-border-router:avahi-control avahi:avahi-control
-> sudo snap connect openthread-border-router:bluez bluez:service
-> ```
-
-## Run
-Start once:
+On Ubuntu Core the `avahi-control` and `bluez` interfaces are not provided by the system.
+These interfaces should be consumed from other snaps, such as the [Avahi](https://snapcraft.io/avahi) and [BlueZ](https://snapcraft.io/bluez) snaps.
+To install these snaps, and establish connections for the `avahi-control` interface from the `avahi` snap, and the `service` interface from the `bluez` snap, run:
+
 ```bash
-sudo snap start openthread-border-router
+sudo snap install avahi bluez
+sudo snap connect openthread-border-router:avahi-control avahi:avahi-control
+sudo snap connect openthread-border-router:bluez bluez:service
 ```
-Add `--enable` flag to make the service start on boot as well.
 
-To query and follow the logs: `snap logs -n 10 -f openthread-border-router`
+## Advanced Usage
```

Comment:
I suggest removing this whole section and migrating the following to [this section](https://canonical-matter.readthedocs-hosted.com/en/latest/how-to/otbr-on-ubuntu/#start-otbr) of the documentation:
```
By default, the services are disabled and not started.
```
```
To start and enable via a [Gadget snap](https://snapcraft.io/docs/the-gadget-snap), set `autostart` to `true`.
```

Reply:
https://github.com/canonical/matter-docs/pull/40